vms/proposervm/post_fork_block.go

```diff
@@ -37,9 +38,20 @@ func (b *postForkBlock) Accept(ctx context.Context) error {
 	if b.slot != nil {
 		b.vm.acceptedBlocksSlotHistogram.Observe(float64(*b.slot))
 	}
+	b.updateLastAcceptedTimestampMetric(outerBlockTypeMetricLabel, b.Timestamp())
+	b.updateLastAcceptedTimestampMetric(innerBlockTypeMetricLabel, b.innerBlk.Timestamp())
 	return nil
 }
 
+const (
+	innerBlockTypeMetricLabel = "inner"
+	outerBlockTypeMetricLabel = "proposervm"
+)
+
+func (b *postForkBlock) updateLastAcceptedTimestampMetric(blockTypeLabel string, t time.Time) {
+	b.vm.lastAcceptedTimestampGaugeVec.WithLabelValues(blockTypeLabel).Set(float64(t.Unix()))
```

Comment:
nit that line is pretty long (currently on half a laptop monitor, that's probably why 😄), maybe split with

```suggestion
	g := b.vm.lastAcceptedTimestampGaugeVec
	g.WithLabelValues(blockTypeLabel)
	g.Set(float64(t.Unix()))
```